requirements.txt

```diff
@@ -1,9 +1,10 @@
-torch==2.4.0
-torchaudio==2.4.0
+torch>=2.0.0
+torchaudio>=2.0.0
```

Comment:
Please change to >=2.4.0 this is inline with silentcipher and should avoid confusion